File: src/net/eduard/permissions/command/PermissionsGroupCreateCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsGroup;
import net.eduard.permissions.manager.PermissionsManager;

public class PermissionsGroupCreateCommand extends CommandManager {

	public PermissionsGroupCreateCommand() {
		super("create", "criar");
		setUsage("/permissions group create <name>");
		setDescription("Criar um grupo de permissões");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 3) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			PermissionsGroup group = manager.getGroup(nome);
			if (group == null) {
				group = manager.createGroup(nome, "", "");
				sender.sendMessage(
						EduPermissions.getInstance().message("group-created").replace("$group", "" + nome));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("group-already-exists").replace("$group", "" + nome));
			}
//			
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsGroupCreateCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsGroupCreateCommand :
    Command("create", "criar") {

    override fun onCommand(sender: Sender , args: List<String>) {
        val manager = PermissionsAPI.instance
        if (args.size < 3) {
            sendUsage(sender)
            return
        }
        val nome = args[2]
        var group = manager.getGroup(nome)
        if (group == null) {
            group = manager.createGroup(nome, "", "")
            sender.sendMessage(
                PermsMessages.message("group-created")
                    .replace("\$group", "" + nome)
            )
        } else {
            sender.sendMessage(
                PermsMessages.message("group-already-exists")
                    .replace("\$group", "" + nome)
            )
        }

    }


    init {
        usage = "/permissions group create <name>"
        description = "Criar um grupo de permissões"
    }
}